### PR TITLE
support asyncpg after 0.29.0 upgrade

### LINKED ISSUE #10: AttributeError with asyncpg v0.29.0
A new release of asyncpg has just been published. It looks like the attribute `asyncpg.Pool._working_addr` does not exist any more (https://github.com/MagicStack/asyncpg/commit/4ddb039746a99b31cc4724329dd600b87a3f9469). But this attribute is expected in the method `hasql.PoolManager.host` (https://github.com/aiokitchen/hasql/blob/ecbf8576a0aa9f8e958b53b5a4f0a6f1736bf54a/hasql/asyncpg.py#L45C1-L47). This results in an `AttributeError` when using the latest version of asyncpg.

```
    def host(self, pool: asyncpg.Pool):
>       addr, _ = pool._working_addr
E       AttributeError: 'Pool' object has no attribute '_working_addr'
```

### PR DESCRIPTION
**Pull request by YiuRULE**
Closes #10

### CODE REVIEW

File: hasql/asyncpg.py [hunk 1]
```diff
@@ -1,15 +1,46 @@
...
+                        BasePoolManager)
 from hasql.metrics import DriverMetrics
 from hasql.utils import Dsn
 
 
 class PoolManager(BasePoolManager):
     pools: Sequence[asyncpg.Pool]
+    cached_hosts: Dict[int, str]
+
+    def __init__(
```
Comment (Pavkazzz) [inline hasql/asyncpg.py:20]: You can make cached_hosts as class var, and doesn't copy all constructor args and kwargs from BasePoolManager

Comment (ns-gtassery) [reply hasql/asyncpg.py]: done

Comment (YiuRULE) [reply hasql/asyncpg.py]: Done, it has been fixed on the latest revision

**Review by mosquito — APPROVED** [review]

**Comment by mosquito** [comment]
@YiuRULE thank you for your contribution. the [`hasql==0.7.0`](https://pypi.org/project/hasql/0.7.0/) already on pypi.

**Pull request merged**